### ISSUE TITLE
pylint reports that os has no member sched_yield

### ISSUE THREAD

**Issue by edreamleo** [pylint-dev/pylint#9946]
### Bug description

Given this program:

```python
"""A quirp: pylint does not know that sched_yield is part of the os module."""

import os

def test_os_sched_yield():
    """pylint reports that os.sched_yield does not exist."""
    os.sched_yield()

pylint reports: `Module 'os' has no 'sched_yield' member`
```

### Command used

```shell
python -m pylint c:\test\test-pylint\test_pylint_os_shed_yield.py
```

### Pylint output

```python
No config file found, using default configuration
************* Module test_pylint_os_shed_yield
C:\test\test-pylint\test_pylint_os_shed_yield.py:11:4: E1101: Module 'os' has no 'sched_yield' member (no-member)
```

### Expected behavior

No error.

### Pylint version

```shell
pylint 3.2.7
astroid 3.2.4
Python 3.12.0 (tags/v3.12.0:0fb18b0, Oct  2 2023, 13:03:39) [MSC v.1935 64 bit (AMD64)]
```

### OS / Environment

```console
Python 3.12.0, PyQt version 6.7.2
Windows 11 AMD64 (build 10.0.22631) SP0
```

**Comment by akamat10** [comment]
I think this is working as expected as the method isn't defined for Windows. See [here](https://github.com/python/typeshed/blob/bde71c575f33fddcb71bbe0550facbfd08b5a6ae/stdlib/os/__init__.pyi#L1050).

**Comment by edreamleo** [comment]
@akamat10 Thanks for your investigation. I agree that the pylint warning is valid. This is a buglet in Leo.

The [python os module page](https://docs.python.org/3/library/os.html#interface-to-the-scheduler) says "[the interface to the scheduler] are only available on some Unix platforms", so I'll close this issue.
